analyzer/src/main/kotlin/managers/SpdxDocumentFile.kt

```diff
@@ -190,15 +192,38 @@ class SpdxDocumentFile(
             Package(
                 id = pkg.toIdentifier(),
                 declaredLicenses = sortedSetOf(pkg.licenseDeclared),
-                concludedLicense = pkg.licenseConcluded.toSpdx(),
+                concludedLicense = buildConcludedLicense(pkg),
                 description = packageDescription,
                 homepageUrl = pkg.homepage.mapNotPresentToEmpty(),
                 binaryArtifact = RemoteArtifact.EMPTY, // TODO: Use "downloadLocation" or "externalRefs"?
-                sourceArtifact = RemoteArtifact.EMPTY, // TODO: Use "sourceInfo" or "externalRefs"?
+                sourceArtifact = buildSourceArtifact(pkg), // TODO: Use "sourceInfo" or "externalRefs"?
                 vcs = VersionControlSystem.forDirectory(packageDir)?.getInfo() ?: VcsInfo.EMPTY
             )
         }
 
         return listOf(ProjectAnalyzerResult(project, packages))
     }
+
+    /**
+     * Return null in case of "NONE" or "NOASSERTION" to fit with the standard process ( concluded manage by curations )
```

Comment:
I believe a better documentation would be:

"Return the concluded license to be used in ORT's data model, which expects a not present value to be null instead of NONE or NOASSERTION."